Add docs for element call external conference configuration (single-page app URL) (#27101)
element-hq/element-web: docs/config.md

      this number is exceeded, the user will not be able to join a given call.
    - `brand`: Optional name for the app. Defaults to `Element Call`. This is
      used throughout the application in various strings/locations.

>     - `guest_spa_url`: Optional url for an Element Call SPA URL. If this is set Element Web will expose the option to share a join

Reviewer: a url for a [...] URL?

      link to public video rooms which can be shared to non matrix users.

>       The guest SPA typically is setup to use a homeserver that allows users to register without email ("passwordless guest users") and to federate.

Reviewer: could you wrap this consistently? A 150-char line followed by a 70-char line followed by another 150-char line seems like the worst of all possible worlds. (Note that the linebreaks do not appear in the [rendered output](https://github.com/element-hq/element-web/blob/d1d909105f4c97da3d90d6c5b8f2e815def88fbb/docs/config.md)

Reviewer: Some minor wording clarifications. I hope I haven't mangled it too badly.

(I assume I'm correct in assuming that it is *not* the "option to share a join link" that is shared, but rather that EW lets you get hold of a link that you then share yourself?)

```suggestion
    - `guest_spa_url`: Optional URL for an Element Call single-page app (SPA), for guest links. If this is set, Element Web will expose the a "join"
      link for public video rooms, which can then be shared to non-matrix users.
      The target Element Call SPA is typically set up to use a homeserver that allows users to register without email ("passwordless guest users") and to federate.
```
